Sources/JWTKit/JWTKeyCollection.swift

```diff
@@ -0,0 +1,239 @@
+import Foundation
+
+/// A collection of JWT and JWK signers for handling JSON Web Tokens (JWTs).
+///
+/// This class provides methods to manage multiple signers, allowing the addition and retrieval of ``JWTSigner`` and ``JWKSigner`` instances.
```

Comment:
Nit this is an actor not class

We should also tidy up the language as we don't expose signers anymore